UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.SubscriptionService.UnitTests/Services/SubscriptionServiceDataTests.cs

```diff
@@ -31,10 +32,10 @@ public async Task WhenCreateOrUpdateSubscriptionThenCreateSubscription()
             CancellationToken cancellationToken = CancellationToken.None;
             SubscriptionRequestMessage subscriptionRequestMessage = GetSubscriptionRequestMessage();
 
-            string response = await _subscriptionServiceData.CreateOrUpdateSubscription(subscriptionRequestMessage, cancellationToken);
+            EventSubscription response = await _subscriptionServiceData.CreateOrUpdateSubscription(subscriptionRequestMessage, cancellationToken);
 
-            A.CallTo(() => _fakeAzureEventGridDomainService.CreateOrUpdateSubscription(subscriptionRequestMessage, cancellationToken)).MustHaveHappenedOnceExactly();
-            Assert.IsInstanceOf<string>(response);
+            A.CallTo(() => _fakeAzureEventGridDomainService.CreateOrUpdateSubscription(subscriptionRequestMessage, A<CancellationToken>.Ignored)).MustHaveHappenedOnceExactly();
```

Comment:
Please pass cancellationToken instead of  A<CancellationToken>.Ignored

Reply:
did changes accordingly